translib/api_tests_app.go

```diff
@@ -0,0 +1,188 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Why is this file and other *_test.go are not in a separate subdir.

Reply:
This is how go works. All test code for a package should be in the package directory itself. Go compiler (`go build` or `go install`) automatically excludes `*_test.go` files. They are used only during `go test`.